types/lodash/lodash-tests.ts

```diff
@@ -5335,43 +5347,63 @@ namespace TestForEach {
     }
 
     {
-        let result: _.List<AbcObject>;
-
-        result = _.forEach(list, (value, index, collection) => {
+        const result = _.forEach(list, (value, index, collection) => {
             value; // $ExpectType AbcObject
             index; // $ExpectType number
             collection; // $ExpectType ArrayLike<AbcObject>
         });
+
+        result; // $ExpectType ArrayLike<AbcObject>
```

Comment:
To simplify this, you should be able to put the `$ExpectType` above `_.forEach`:

```ts
// $ExpectType ArrayLike<AbcObject>
_.forEach(list, (value, index, collection) => {
    value; // $ExpectType AbcObject
    index; // $ExpectType number
    collection; // $ExpectType ArrayLike<AbcObject>
});
```

Reply:
I figured that out later in the mapValues tests :). Forgot to go back and simplify these tests.